XenModel/Messages.resx

```diff
@@ -8047,6 +8047,9 @@ You should only proceed if you have verified that these settings are correct.</v
   <data name="NEWPOOL_MASTER_ROLE" xml:space="preserve">
     <value>Your current role on the master is not authorized to add hosts to the master's pool</value>
   </data>
+  <data name="NEWPOOL_MAX_NUMBER_HOST_REACHED" xml:space="preserve">

```

Comment:
The message should say something on the lines of: the pool already reached the maximum number of servers allowed by your license